app/views/devise/invitations/edit.html.erb

```diff
@@ -1,24 +1,24 @@
-<h2><%= t 'devise.invitations.edit.header' %></h2>
-
-<%= form_for resource, :as => resource_name, :url => invitation_path(resource_name), :html => { :method => :put } do |f| %>
-    <%= devise_error_messages! %>
-    <%= f.hidden_field :invitation_token %>
-
-    <%= f.label 'First Name' %>
-    <%= f.text_field :first_name, class: 'input' %>
-
-    <%= f.label 'Last Name' %>
-    <%= f.text_field :last_name, class: 'input' %>
-
-    <%= f.label 'Phone Number' %>
-    <%= f.text_field :phone, class: 'input' %>
-
-  <% if f.object.class.require_password_on_accepting %>
-      <%= f.label :password %>
-      <%= f.password_field :password, class: 'input' %>
-      <%= f.label :password_confirmation %>
-      <%= f.password_field :password_confirmation, class: 'input' %>
-  <% end %>
-
-  <p><%= f.submit t("devise.invitations.edit.submit_button"), class: 'button-primary' %></p>
-<% end %>
+<div class="info-block-wrapper centered">
+    <div class="info-block">
+        <h2><%= t 'devise.invitations.edit.header' %></h2>
+        <%= form_for resource, :as => resource_name, :url => invitation_path(resource_name), :html => { :method => :put } do |f| %>
+            <%= devise_error_messages! %>
+            <%= f.hidden_field :invitation_token %>
+            <%= f.label 'First Name' %>
```

Comment:
Have this text field in focus when first enter page so user knows where to start. All other forms have this so we should be consistent

Reply:
I'lI look into this and figure it out!